File: app/src/main/java/com/eiyooooo/foldswitcher/MyApplication.java
package com.eiyooooo.foldswitcher;

import android.app.Application;
import android.content.Context;

import org.lsposed.hiddenapibypass.HiddenApiBypass;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
    }

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        HiddenApiBypass.addHiddenApiExemptions("L");
    }
}

File: app/src/main/java/com/eiyooooo/foldswitcher/MyApplication.kt
package com.eiyooooo.foldswitcher

import android.app.Application
import android.content.Context
import com.eiyooooo.foldswitcher.helpers.SharedPreferencesHelper
import com.google.android.material.color.DynamicColors
import org.lsposed.hiddenapibypass.HiddenApiBypass

class MyApplication : Application() {
    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        HiddenApiBypass.addHiddenApiExemptions("L")
    }

    override fun onCreate() {
        super.onCreate()
        SharedPreferencesHelper.init(this)
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}